Dockerfile

```diff
@@ -0,0 +1,15 @@
+# build stage
+FROM golang:alpine AS build-env
+RUN apk --no-cache add build-base git bzr mercurial gcc
+ADD . /src
+WORKDIR /src
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+ENV GOARCH=386
```

Comment:
why set these any of these? I'd prefer to have 64-bit builds if we can.

Reply:
Legacy reasons. Correct, not needed here